examples/RF24Mesh_Example/RF24Mesh_Example.ino

```diff
@@ -71,10 +79,10 @@ void loop() {
     if (!mesh.write(&displayTimer, 'M', sizeof(displayTimer))) {
 
       // If a write fails, check connectivity to the mesh network
-      if ( ! mesh.checkConnection() ) {
+      if (!mesh.checkConnection()) {
         //refresh the network address
         Serial.println("Renewing Address");
-        if (!mesh.renewAddress()) {
+        if (mesh.renewAddress() == MESH_DEFAULT_ADDRESS) {
```

Comment:
This is the only instance I could find which improperly uses the return value of `renewAddress()`.